packages/strapi-plugin-content-type-builder/controllers/ContentTypeBuilder.js

```diff
@@ -188,5 +188,24 @@ module.exports = {
     ctx.send({
       autoReload: _.get(strapi.config.environments, 'development.server.autoReload', false),
     });
+  },
+
+  checkTableExists: async ctx => {
+    // Get connec
+    const { connection } = ctx.params;
+    const model = _.toLower(ctx.params.model);
```

Comment:
What happens if the model doesn't exist? You should return an error in this case.

Reply:
In this particular case I shouldn't because I send the request when before you saved your first content type to check whether or not your table exists...